<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

	<arg name="name" 		default="ximu3" />
	<arg name="publish_status"	default="true" />
	<arg name="parent_frame_id" 	default="map" />
	<arg name="ip_address" 		default="192.168.1.12" />
	<arg name="send_port" 		default="8012" />
	<arg name="receive_port" 	default="9012" />
	<arg name="do_calibration" 	default="true" />
	<arg name="use_imu_time_stamps" 	default="true" />
	
	<!-- ahrs_divisor
		0 .. disabled
		1 .. 400 hz
		2 .. 200 hz
		3 .. 133 hz
		4 .. 100 hz
		8 .. 50 hz
	-->

	<arg name="ahrs_divisor" 	default="1" />

	<node pkg="ximu3_ros" type="imu_node" 	name="ximu_$(arg name)" output="screen">
		<param name="name" 		value="$(arg name)" />
		<param name="publish_status" 	value="$(arg publish_status)" />
		<param name="parent_frame_id" 	value="$(arg parent_frame_id)" />
		<param name="receive_port"	value="$(arg receive_port)" />
		<param name="send_port" 	value="$(arg send_port)" />
		<param name="ip_address"	value="$(arg ip_address)" />
		<param name="ahrs_divisor"	value="$(arg ahrs_divisor)" />
		<param name="do_calibration"	value="$(arg do_calibration)" />
		<param name="use_imu_time_stamps"	value="$(arg use_imu_time_stamps)" />
        </node>

</launch>

